src/lib.rs

```diff
@@ -1480,6 +1480,7 @@ impl<T: TensorType> Display for Tensor<T> {
 #[derive(Debug)]
 pub struct Library {
     inner: *mut tf::TF_Library,
+    op_list: protos::op_def::OpList,
```

Comment:
Since this field is private (which is a good thing), there's no way for clients to get the list.  `Library` will need an accessor for clients to get the data, and the test should test against that.

Reply:
Sorry for the delay. Would this accessor want to return the proto object? Or do we want to wrap the OpList (and internal OpDefs) in some new struct to avoid exposing protos?

Reply:
I've been avoiding exposing the protos in the API directly so far, since that would force users to use a specific proto library.  It might still be worth doing at some point for the convenience, but I don't want to do that in the middle of this pull request, at least.  We'll need a new struct to expose the information we need.  We can start with just the basics and add fields later if necessary.

Reply:
I've started by adding an `OpList` and `OpDef` struct to abstract the protos. I tried to follow patterns used elsewhere in the crate, but I want to make sure the design I'm following makes sense before implementing the `OpDef_ArgDef` and `OpDef_AttrDef` structs as well.